core/src/style/style.h

```diff
@@ -36,6 +35,14 @@ enum class Blending : int8_t {
     overlay,
 };
 
+struct StyledMesh {
```

Comment:
Collecting: DrawBucket, Drawable, Batch?

I would say it belongs to Style - it's what the StyleBuilder produces and Style should know how to render them (in my plan)